modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts

```diff
@@ -452,7 +452,7 @@ export interface BaseBroadcastTransactionOptions {
 }
 
 export interface BaseBroadcastTransactionResult {
-  txId: string;
+  txId?: string;
```

Comment:
Few comments here: 
- isn't this a breaking change? We need to update the commit when changes like this happens
- why not add your array of txids here as well, that way other coins can re-use this vs making this sui specific 

```
export type BaseBroadcastTransactionResult  = {
  txId: string;
} | {
txIds: string[]
```

This way it can be a generic type other coins use.

Reply:
@zahin-mohammad changing an optional field to required would be a breaking change.
making a required field optional isn't a breaking change if I'm not wrong.

The suggested code is giving the following error in the IDE:
<img width="747" alt="image" src="https://github.com/user-attachments/assets/6c1c68bf-57ca-4062-b83d-84a087204d0a">

Reply:
@zahin-mohammad I've moved the changes to a new branch because the commit history could not be squashed due to merge commits.
New PR: https://github.com/BitGo/BitGoJS/pull/4838